demos/python_demos/speech_recognition_demo/ctcdecode-numpy/CMakeLists.txt

```
@@ -0,0 +1,11 @@
+# Copyright (C) 2020 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+execute_process(
+COMMAND "${PYTHON_EXECUTABLE}" -m pip install .
```

to make sure that do not install it with python2.7
```suggestion
COMMAND "${PYTHON_EXECUTABLE}" -m pip3 install .
```

---

On both Ubuntu 18.04 and Windows 10, "python -m pip3" does not work, only "pip3" works. I think, ${PYTHON_EXECUTABLE} runs the correct version of python. I think running "${PYTHON_EXECUTABLE} -m pip" should be more robust than running "pip3". OK?

---

ok